google/orbit
# Remove uprobes-based manual instrumentation

In particular, all the code related to "Orbit function types" can be removed.

See notes in the bug why we believe this change is not problematic.

Bug: http://b/194288616

Test:
- Open a capture taken with main on `OrbitTestLegacyApi`: As expected, many
  things regarding manual instrumentation are now wrong, but the capture can
  still be opened and the rest is correct.
- Open a capture taken with main on `OrbitTest`, verify no information was lost.
- Take a capture on `OrbitTest`, verify manual instrumentation.
- Take a capture on `OrbitTest` also with dynamic instrumentation and try
  various filters in the capture view.
